wp-parsely.php

```diff
@@ -281,30 +281,23 @@ function maybe_load_plugin() {
 		Parsely_Loader_Info::set_integration_type( Parsely_Integration_Type::ENABLED_CONSTANT );
 	}
 
-	$option_load_status   = get_option( '_wpvip_parsely_mu', null );
 	$filtered_load_status = apply_filters( 'wpvip_parsely_load_mu', null );
 
 	// If plugin isn't enabled via constant then check for filter and option status.
 	if ( true !== $parsely_enabled_constant ) {
-		$should_load            = true === $filtered_load_status || '1' === $option_load_status;
-		$should_prevent_loading = false === $filtered_load_status || '0' === $option_load_status;
+		$should_load            = true === $filtered_load_status;
+		$should_prevent_loading = false === $filtered_load_status;
```

Comment:
This now reads ... weird. But I didn't want to introduce any extra changes to the logic.

Reply:
I think we can just check directly for the `$filtered_load_status` because it does read weird af now.